naming-workflow-in-javascript-sdk.md

```diff
@@ -155,13 +185,11 @@ const main = async (name) => {
   console.log(update);
 };
 
-main('YOUR_NAME_TO_REGISTER.test');
+main('emmanueljet.test');
```

Comment:
you should await `main`

Reply:
main can't be awaited there

Reply:
@emmanuelJet can you revert to `YOUR_NAME_TO_REGISTER.test`

Reply:
Will do that now

Reply:
@thepiwo `emmanueljet.test` has being reverted to `YOUR_NAME_TO_REGISTER.test`